clients/spark/build.sbt

```diff
@@ -1,6 +1,6 @@
-lazy val projectVersion = "0.15.0"
+lazy val projectVersion = "0.16.0"
 version := projectVersion
-lazy val hadoopVersion = "3.2.1"
+lazy val hadoopVersion = "3.3.6"
```

Comment:
Just to make sure - we compile with a newer version and expect it to run on an older version?
asking as it is usually the other way around.